### DISCUSSION TITLE
[Question] Does increasing sim speed or waypoint jumping (Toliss) break Auto Ortho?

### DISCUSSION THREAD

**Discussion by OceanCode45** [kubilus1/autoortho#357] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Win11
- XPlane Version: XP12
- AutoOrtho Version 0.5.1

**Comment by Jeff2122** [comment]
I actually think it will hose your flight. You will be waiting for downloads often, too often. I have not tried this as I don't use accelerated time, but I see some having slow load issues or blank tiles with fast jets already. Accel would really do it. Try it and let us know, would you?

**Comment by chezhj** [comment]
If you have "show warning missing scenery" enabled in xp, and ao is "late at the table" xp will show a warning and stop. Always fly with scenery warnings disabled to avoid a sudden stop of xplane
